src/Platform.elm

```diff
@@ -63,6 +63,85 @@ Initializing a headless program from JavaScript looks like this:
 ```javascript
 var app = Elm.MyThing.worker();
 ```
+
+### init
+
+The first argument, `init` is a [Tuple](Tuple) that lets you specify how your program should start.
+Its first value is the initial state of your program's model.
+The second value is an initial command. Often that command is [Cmd.none](Platform-Cmd#none).
+
+### update
+
+The second argument, `update` is a function that takes these two arguments:
+
+1. A message from the outside world, that has the information your code will act upon.
+Usually this is a `Msg` type that you specify, (a constructor), which takes a record type
+matching the data coming from an incoming port. Unpack that data with `case` ... `of`.
+
+2. The current state of your program's model, as maintained by the Elm runtime.
+
+The `update` function must return a [Tuple](Tuple)
+containing the modified form of your program's model, and a command.
+Generate that command by passing data to an outbound port. The data type
+held by this command is a determined by Elm. It probably will __not__ match any type
+you have defined in your code. Here's an example where we output a [String](String).
+
+```elm
+port greet : String -> Cmd a
+
+type Msg = Name String
+
+type alias Model = { greetings : Int }
+
```

Comment:
If I include that, then I have an init function, and may as well have a subscriptions function, then may as well have the Process.program call for completeness.